pkg/release/release.go

```diff
@@ -33,3 +42,28 @@ var Image = map[string]string{
 	"pod":                       "quay.io/microshift/pause:" + Base,
 	"service_ca_operator":       "quay.io/microshift/service-ca-operator:" + Base,
 }
+
+func init() {
+	arch_replacer := strings.NewReplacer("amd64", "x86_64", "arm64", "aarch64")
+	arch := arch_replacer.Replace(runtime.GOARCH)
+
+	release_file := "release/release-" + arch + ".json"
+	data, err := embedded.Asset(release_file)
+	if err != nil {
+		// If there is no release file for this architecture, work with the generic specs
+		return
+	}
+
+	var release map[string]any
+	if err := json.Unmarshal(data, &release); err != nil {
+		panic(fmt.Errorf("unmarshaling %s: %v", release_file, err))
+	}
+
+	images := release["images"].(map[string]any)
+	for name, pullspec := range images {
+		name := strings.Replace(name, "-", "_", -1)
+		if _, ok := Image[name]; ok {
+			Image[name] = pullspec.(string)
+		}
+	}
```

Comment:
Can we add some comments in this section on what we're trying to do and why?

Reply:
Done.